x-pack/test/api_integration/apis/ml/system/space_capabilities.ts

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { MlCapabilitiesResponse } from '../../../../../plugins/ml/common/types/capabilities';
+
+const idSpaceWithMl = 'space_with_ml';
+const idSpaceNoMl = 'space_no_ml';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const spacesService = getService('spaces');
+  const ml = getService('ml');
+
+  async function runRequest(user: USER, space?: string): Promise<MlCapabilitiesResponse> {
+    const { body } = await supertest
+      .get(`${space ? `/s/${space}` : ''}/api/ml/ml_capabilities`)
+      .auth(user, ml.securityCommon.getPasswordForUser(user))
+      .set(COMMON_REQUEST_HEADERS)
+      .expect(200);
+
+    return body;
+  }
+
+  describe('ml_capabilities in spaces', () => {
+    before(async () => {
+      await spacesService.create({ id: idSpaceWithMl, name: 'space_one', disabledFeatures: [] });
+      await spacesService.create({ id: idSpaceNoMl, name: 'space_two', disabledFeatures: ['ml'] });
+    });
+
+    after(async () => {
+      await spacesService.delete(idSpaceWithMl);
+      await spacesService.delete(idSpaceNoMl);
+      await ml.api.cleanMlIndices();
+      await ml.testResources.cleanMLSavedObjects();
```

Comment:
It doesn't hurt to clean this up, but it's not needed as we don't create anything to cleanup there in this test suite.